packages/url/README.md

```diff
@@ -96,11 +96,13 @@ _Usage_
 
 ```js
 const displayUrl = filterURLForDisplay( 'https://www.wordpress.org/gutenberg/' ); // wordpress.org/gutenberg
+const imageUrl = filterURLForDisplay( 'https://www.wordpress.org/wp-content/uploads/img.png', 20 ); // …ent/uploads/img.png
 ```
 
 _Parameters_
 
 -   _url_ `string`: Original URL.
+-   _maxLength_ `(number|null)`: URL length.
```

Comment:
This adds changes to the public API of `@wordpress/url` npm package. It is always a good idea to add an entry in the CHANGELOG so folks consuming this package could learn about new capabilities. How about we add a note in:

https://github.com/WordPress/gutenberg/blob/master/packages/url/CHANGELOG.md

Reply:
@gziolo Thank you for the recommendation. Updated the changelog in https://github.com/WordPress/gutenberg/pull/27530/commits/1330576d52dc5302d5bfbdd675df12770b00b4d5 and rebased this PR.

Reply:
Great, thank you!